db/migrate/20211105142202_queue_notification_update_mail.rb

```diff
@@ -0,0 +1,13 @@
+class QueueNotificationUpdateMail < ActiveRecord::Migration[6.1]
+  def up
+    User.active.find_each do |user|
+      AnnouncementMailer
+        .announce(user,
+                  subject: I18n.t(:'notifications.update_info_mail.subject'),
+                  body: I18n.t(:'notifications.update_info_mail.body'),
+                  body_header: I18n.t(:'notifications.update_info_mail.body_header'),
+                  body_subheader: I18n.t(:'notifications.update_info_mail.body_subheader'))
+        .deliver_later
```

Comment:
This segment takes 92 seconds to migrate on a community dump. I'm fine with merging it as is but we might want to skip this migration then and do it later (/cc @machisuji) , or we turn this into something like an `AnnouncementJob`  that:

1. creates one job and then mails inline without another job (individual mails might fail more easily then)
2. creates the mailer jobs separately.

Reply:
After corresponding with Markus, I added https://github.com/opf/openproject/pull/9815 to do option 2